projects/go-lib/src/lib/animations/table-details.animation.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  animate,
+  AnimationTriggerMetadata,
+  state,
+  style,
+  transition,
+  trigger
+} from '@angular/animations';
+
+import { easing, timing } from './_configs';
+
+export const detailButtonAnim: AnimationTriggerMetadata = trigger('detailButtonAnim', [
+  state('open', style({
+    transform: 'rotate(90deg)'
+  })),
+  state('close', style({
+    transform: 'rotate(0)'
+  })),
+  transition('open <=> close', [
+    animate(timing + easing)
+  ])
+]);
+
+export const detailsAnimation: AnimationTriggerMetadata = trigger('detailsAnimation', [
+  state('open', style({
+    display: 'table-row'
+  })),
+  state('close', style({
+    display: 'none'
+  })),
+  transition('open <=> close', [
+    animate('.125s ' + easing)
```

Comment:
Do you feel like we need to have a different timing for this?

Reply:
Also, you can't perform an animation on `display` so this doesn't really do much visually.